multiqc/modules/quast/quast.py

```diff
@@ -53,7 +53,12 @@ def __init__(self):
             'anchor': 'quast-contigs',
             'content': self.quast_contigs_barplot()
         })
-
+        # Number of genes plot
+        self.sections.append({
```

Comment:
Please only add this section if the predicted genes are found in the QUAST output. When I run this code I get an empty header + nav link in the report with no content.